yb-voyager/main.go

```diff
@@ -35,6 +35,7 @@ func main() {
 	registerSignalHandlers()
 	atexit.Register(cmd.PackAndSendCallhomePayloadOnExit)
 	atexit.Register(cmd.CleanupChildProcesses)
+	atexit.Register(restoreTerminalState) // ensure terminal is always restored
```

Comment:
Implementation of `restoreTerminalState` has a call to `utils.ErrExit`, which will call `atexit.Exit()` potentially causing an infinite loop! 
Exit handlers should be very light. See if you can replace the call to utils.ErrExit with just a log perhaps?

Reply:
yeah log.Errorf() or log.Fatalf() should be fine. Will test and do the change.

Reply:
Great catch!! I manually tested, utils.ErrExit() was getting into a never-ending loop.

With log.Fatalf() it will just log the error and exit. 
But nothing will be printed on the console.

 I am thinking to go with log.Errorf(), since this err won't fail the migration, just the terminal stdin/stdout will be in wrong state.

Reply:
Makes sense! Just logging the error and moving on should be fine